src/Sarif.Multitool/RebaseUriCommand.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.CodeAnalysis.Sarif.Processors;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    internal static class RebaseUriCommand
+    {
+        public static int Run(RebaseUriOptions rebaseOptions)
+        {
+            Uri baseUri;
+            if (!Uri.TryCreate(rebaseOptions.BasePath, UriKind.Absolute, out baseUri))
+            {
+                Console.WriteLine($"BasePath {rebaseOptions.BasePath} was not an absolute URI.  It must be.");
+                return -1;
```

Comment:
> return -1; [](start = 15, length = 11)

your result handling is diverging from other commands. check out the RewriteCommand. I don't mind altering the convention in that command but let's make sure all of them stay consistent.

The RewriteCommand would handle this like so, all exits are accomplished by throwing an exception. the core command wraps catch(Exception ex), writes ex.ToString() to the console, and returns 1 to indicate an error.